clippy.toml

```diff
@@ -0,0 +1,2 @@
+large-error-threshold = 256
+
```

Comment:
Extra line?

Reply:
It has 2 lines only, typically github doesn't show the last like this?